app/connectors/GetBusinessDetailsConnector.scala

```diff
@@ -30,13 +30,19 @@ class GetBusinessDetailsConnector @Inject()(val http: HttpClient,
                                             val appConfig: MicroserviceAppConfig
                                            )(implicit ec: ExecutionContext) extends RawResponseReads {
 
-  def getBusinessDetailsUrl(nino: String): String = s"${appConfig.ifUrl}/registration/business-details/nino/$nino"
+  def getUrl(accessType: BusinessDetailsAccessType, ninoOrMtdRef: String) = {
+    accessType match {
+      case Nino => s"${appConfig.ifUrl}/registration/business-details/nino/$ninoOrMtdRef"
```

Comment:
Why call it ninoOrMtdRef rather than ninoOrMtdId?

Reply:
The previous variables were called mtdRef, despite the API spec and url using MtdId, so thought I'd preserve it. Might be an old name tho